launchable/commands/record/session.py

```diff
@@ -35,8 +35,15 @@
     cls=KeyValueType,
     multiple=True,
 )
+@click.option(
+    "--observation",
+    "is_observation",
+    help='observation',
```

Comment:
[question]
Is it obvious for user what `--observation` is used for with help only saying `observation`?

Reply:
Ops, it's still a draft... I'll fix it. Thanks